matplotcheck/tests/test_notebook.py

```diff
@@ -0,0 +1,145 @@
+"""Tests for the notebook module"""
+
+import pytest
+import matplotlib.pyplot as plt
+from matplotcheck.base import PlotTester
+import matplotcheck.notebook as nb
+
+
+@pytest.fixture
+def locals_dictionary_good():
+    """Abriged locals dictionary of a jupyter notebook with import in the proper
+    place"""
+    return {
+        "__name__": "__main__",
+        "__doc__": "Automatically created module for IPython interactive",
+        "_ih": ["", "import os", 'print("Good notebook")', "locals()"],
+        "In": ["", "import os", 'print("Good notebook")', "locals()"],
+        "Out": {},
+        "_i": 'print("Good notebook")',
+        "_ii": "import os",
+        "_iii": "",
+        "_i1": "import os",
+        "_i2": 'print("Good notebook")',
+        "_i3": "locals()",
+    }
+
+
+@pytest.fixture
+def locals_dictionary_bad():
+    """Abriged locals dictionary of a jupyter notebook with import in the wrong
+    place"""
+    return {
+        "__name__": "__main__",
+        "__doc__": "Automatically created module for IPython interactive",
+        "_ih": ["", 'print("Bad notebook")', "import os", "locals()"],
+        "In": ["", 'print("Bad notebook")', "import os", "locals()"],
+        "Out": {},
+        "_i": "import os",
+        "_ii": 'print("Bad notebook")',
+        "_iii": "",
+        "_i1": 'print("Bad notebook")',
+        "_i2": "import os",
+        "_i3": "locals()",
+    }
+
+
+def test_notebook_convert_single_axes(basic_polygon_gdf):
+    """Test convert single axes."""
```

Comment:
Test that convert_axes returns the correct matplotlib axis object?

is that what this is doing? if so let's make the docstring really clear.